federation-js/src/composition/validate/preComposition/__tests__/duplicateEnumOrScalar.test.ts

```diff
@@ -60,13 +62,19 @@ describe('duplicateEnumOrScalar', () => {
 
     const warnings = validateDuplicateEnumOrScalar(serviceA);
     expect(warnings).toMatchInlineSnapshot(`
-            Array [
-              Object {
-                "code": "DUPLICATE_ENUM_DEFINITION",
-                "message": "[serviceA] ProductType -> The enum, \`ProductType\` was defined multiple times in this service. Remove one of the definitions for \`ProductType\`",
-              },
-            ]
-        `);
+      Array [
+        Object {
+          "code": "DUPLICATE_ENUM_DEFINITION",
+          "locations": Array [
```

Comment:
Should we provide the locations of all enums that have a name collision in a single service?

This is such an odd composition error IMO, seems like this would be handled in `graphql-js`. I wonder what caused us to build this. (ignore me though, just noting / thinking out loud)

Reply:
I am going to punt on this as part of the effort for #705, since we don't know which service to associate those locations with right now